src/components/negotiations-panel.tsx

```diff
@@ -58,8 +58,7 @@ export interface NegotiationsPanelProps {
   negotiationStatusUpdateTS: number; // used to trigger rerenders
 }
 
-
-export const NegotiationsPanel = React.memo((props: NegotiationsPanelProps) => {
+export default function NegotiationsPanel(props: NegotiationsPanelProps): JSX.Element {
```

Comment:
Maybe I'm missing something or maybe is a rebase mistake. But you're removing the memo here (which improves performance). 

You can check the difference on `src/components/tests/__snapshots__/negotiations-panel.test.tsx.snap` too

Related PR https://github.com/osrf/romi-dashboard/pull/112

Reply:
nice catch cd53210